src/Solvers/conjugate_gradient_poisson_solver.jl

```diff
@@ -30,9 +31,15 @@ function Base.show(io::IO, ips::ConjugateGradientPoissonSolver)
               "    └── iteration: ", prettysummary(ips.conjugate_gradient_solver.iteration))
 end
 
+@inline function V∇²ᶜᶜᶜ(i, j, k, grid, c)
+    return δxᶜᶜᶜ(i, j, k, grid, Ax_∂xᶠᶜᶜ, c) +
+           δyᶜᶜᶜ(i, j, k, grid, Ay_∂yᶜᶠᶜ, c) +
+           δzᶜᶜᶜ(i, j, k, grid, Az_∂zᶜᶜᶠ, c)
+end
+
 @kernel function laplacian!(∇²ϕ, grid, ϕ)
```

Comment:
This is not just the Laplacian anymore, right? How about

```suggestion
@kernel function _symmetric_laplacian_operator!(∇²ϕ, grid, ϕ)
```